mcs/class/System/ReferenceSources/AutoWebProxyScriptEngine.cs

```diff
@@ -153,6 +164,56 @@ WebProxyData InitializeRegistryGlobalProxy ()
 			return null;
 		}
 
+		WebProxyData InitializeProxyDataUWP()
```

Comment:
This proves I need to read my diffs before commiting. /facepalm. I'll remove this, it was a WIP copy-pasta.